package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "license": "AGPL-3.0-only",
   "dependencies": {
-    "@graasp/query-client": "git://github.com/graasp/graasp-query-client.git#main",
-    "@graasp/ui": "git://github.com/graasp/graasp-ui.git#master",
+    "@graasp/query-client": "git://github.com/graasp/graasp-query-client.git#yarn2",
```

Comment:
Will we update both main branches for `graasp-query-client` and `graasp-ui` and point to them before merging?

Reply:
Yes of course, I'll create the PRs for them right now. They will be merged before this one.